gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -66,6 +66,7 @@
     private static final String PROP_ERROR_PRONE_APPLY = "errorProneApply";
     private static final String PROP_REFASTER_APPLY = "refasterApply";
     private static final String DISABLE_PROPERTY = "com.palantir.baseline-error-prone.disable";
+    private static final String ENABLE_PROPERTY = "com.palantir.baseline-error-prone.enable";
```

Comment:
seems a tiny bit funky to have two flags here, how would you feel about using it as 'key=value' manner rather than just testing for the presence of 'key'?

I'm thinking we'd just have the `com.palantir.error-prone.disable` flag and say it defaults to be `runningInIntellij`, but if you wanted to provide it explicitly you can??